### PR TITLE
[16.0][MIG] account_invoice_section_sale_order

### PR DESCRIPTION
**Pull request by rven**

### CODE REVIEW

**Comment by tv-openbig** [comment]
Functional Review: OK
Bildschirmfoto 2023-04-28 um 18 05 05

**Review by andreampiovesana — APPROVED** [review]
LGTM

**Review by FrancoMaxime — APPROVED** [review]
LGTM

File: account_invoice_section_sale_order/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    _inherit = "sale.order"
+
+    def _create_invoices(self, grouped=False, final=False, date=None):
+        """Add sections by sale order in the invoice line.
+
+        Order the invoicing lines by sale order and add lines section with
+        the sale order name.
+        Only do this for invoices targetting multiple sale order
+        """
+        invoice_ids = super()._create_invoices(grouped=grouped, final=final)
```
Comment (hildickethan) [inline account_invoice_section_sale_order/models/sale_order.py]:
```suggestion
        invoice_ids = super()._create_invoices(grouped=grouped, final=final, date=date)
```

**Review by hildickethan — APPROVED** [review]

**Review by Gelo-fl — APPROVED** [review]
LGTM https://www.awesomescreenshot.com/video/18275318?key=921edd53db10dc30343873cec68720b9

**Pull request merged**